sample-apps/javascript-sample-app/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "dependencies": {
+    "@opentelemetry/api": "^1.2.0",
+    "@opentelemetry/exporter-collector-grpc": "^0.25.0",
+    "@opentelemetry/id-generator-aws-xray": "^0.25.0",
+    "@opentelemetry/instrumentation": "^0.25.0",
+    "@opentelemetry/instrumentation-http": "^0.25.0",
+    "@opentelemetry/metrics": "^0.24.0",
+    "@opentelemetry/propagator-aws-xray": "^0.25.0",
+    "@opentelemetry/sdk-metrics-base": "^0.25.0",
```

Comment:
why we are not using the latest dependencies? https://github.com/open-telemetry/opentelemetry-js/tree/main/experimental/packages/opentelemetry-sdk-metrics

Reply:
Currently the latest dependencies break metrics. A future patch will be pushed out to update the dependencies as well as the app itself.